src/adam/casadi/computations.py

```diff
@@ -28,12 +29,11 @@ def __init__(
             joints_name_list (list): list of the actuated joints
             root_link (str, optional): the first link. Defaults to 'root_link'.
         """
-        super().__init__(
-            urdfstring=urdfstring,
-            joints_name_list=joints_name_list,
-            root_link=root_link,
-            gravity=gravity,
-        )
+        factory = URDFModelFactory(urdfstring, SpatialMath())
+        model = Model.build(factory=factory, joints_name_list=joints_name_list)
+        self.rbdalgos = RBDAlgorithms(model=model, math=SpatialMath())
```

Comment:
I think it's better to create only one `SpatialMath` (same for the other implementations):

```suggestion
        math = SpatialMath()
        factory = URDFModelFactory(urdfstring, math)
        model = Model.build(factory=factory, joints_name_list=joints_name_list)
        self.rbdalgos = RBDAlgorithms(model=model, math=math)
```